systemtest/src/main/java/io/strimzi/systemtest/kafkaclients/internalClients/BridgeClients.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.kafkaclients.internalClients;
+
+import io.fabric8.kubernetes.api.model.LocalObjectReference;
+import io.fabric8.kubernetes.api.model.PodSpecBuilder;
+import io.fabric8.kubernetes.api.model.batch.v1.JobBuilder;
+import io.strimzi.systemtest.Constants;
+import io.strimzi.systemtest.Environment;
+import io.sundr.builder.annotations.Buildable;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Buildable(editableEnabled = false)
+public class BridgeClients extends KafkaClients {
+    private int port;
+    private int pollInterval;
+
+    public int getPollInterval() {
+        return pollInterval;
+    }
+
+    public void setPollInterval(int pollInterval) {
+        this.pollInterval = pollInterval;
+    }
+
+    public int getPort() {
+        return port;
+    }
+
+    public void setPort(int port) {
+        this.port = port;
+    }
+
+    public JobBuilder producerStrimziBridge() {
```

Comment:
I wasn't clear to me before if we really needed to be returning `JobBuilder` and not `Job` all the call sites except one that I saw just called `build` on the `JobBuilder` returned from these APIs. For the one exception we could either just `new JobBuilder(producerStrimziBridge())` and edit the things we needed, or possibly isn't it a bit of an indication that our builders weren't sufficiently expressive that we needed to go changing the Job returned from our builder in that one call site?

Reply:
Yeah that's right, I guess we kept it to be `JobBuilder` for few occurrences, but now it's just one, which I resolved with `new JobBuilder(..)`. Good catch! :)